posts/2024/2024_07_29_kaustav_week_9.rst

```diff
@@ -0,0 +1,65 @@
+My Journey Continues: Week 9 Progress with DIPY
+===============================================
+
+.. post:: July 29, 2024
+   :author: Kaustav
+   :tags: google
+   :category: gsoc
+
+Hello everyone! It’s time for another update on my progress. Week 9 has been a blend of learning, preparation, and a bit of personal work as I continue my journey with the dipy.org project. This week, I focused on diving into Docker, an essential tool for the upcoming tasks in our project. Let me take you through what I accomplished and learned over the past few days.
+
+Learning Docker
+---------------
+
+One of the key goals for this week was to get up to speed with Docker, as it will play a significant role in the next phase of our project. I’ve heard a lot about Docker’s versatility and how it simplifies the process of managing development environments, so I was eager to dive in.
+
+**Getting Started with Docker**
+
+I started my Docker journey by exploring the official `Docker documentation <https://docs.docker.com/get-started/>`_, which provided a solid foundation. The tutorials and examples there were incredibly helpful in understanding the basics of Docker, including concepts like containers, images, Dockerfiles, and volumes. The step-by-step guide walked me through setting up my first container, which was a satisfying experience.
+
+**Trial and Error**
+
+While the documentation gave me a good start, much of my learning came through trial and error. I experimented with creating and running containers, exploring how to build custom Docker images, and understanding how to manage multiple containers simultaneously.
+
+I also explored some video tutorials on YouTube.
+
+1. https://www.youtube.com/watch?v=pTFZFxd4hOI&t=551s
```

Comment:
You could use a Sphinx "inline external link" to reference tutorial links like you did at [L18].

`Docker Tutorial for Beginners <https://www.youtube.com/watch?v=pTFZFxd4hOI&t=551s>`_